### PR TITLE
Add ingress healthz

### PR DESCRIPTION
**Pull request by phillebaba**
Adds ingress healthz as a module to aks-core.

### CODE REVIEW

File: modules/kubernetes/ingress-healthz/templates/values.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+application:
+  environment: ${environment}
+
```
Comment (phillebaba) [inline modules/kubernetes/ingress-healthz/templates/values.yaml.tpl:3]: need a good way of setting the domain and certificate issuer.

**Review by simongottschlag — CHANGES_REQUESTED** [review]
We need to add ingress to the helm chart before merging this

**Comment by NissesSenap** [comment]
Can you explain what you mean @simongottschlag "We need to add ingress to the helm chart before merging this"
There is a ingress template in the helm chart.
https://github.com/XenitAB/ingress-healthz/blob/master/charts/ingress-healthz/templates/ingress.yaml

File: modules/kubernetes/ingress-healthz/templates/values.yaml.tpl (hunk 1 above)
Comment (NissesSenap) [reply modules/kubernetes/ingress-healthz/templates/values.yaml.tpl:3]: Yeah other then that I think it looks good.

**Comment by simongottschlag** [comment]
> Can you explain what you mean @simongottschlag "We need to add ingress to the helm chart before merging this"
> There is a ingress template in the helm chart.
> https://github.com/XenitAB/ingress-healthz/blob/master/charts/ingress-healthz/templates/ingress.yaml

never mind :-)

**Review by simongottschlag — APPROVED** [review]
LGTM

**Pull request merged**